src/Microsoft.ML.Data/DataLoadSave/FakeSchema.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.ML.Data.DataLoadSave
+{
+
+    /// <summary>
+    /// A fake schema that is manufactured out of a SchemaShape.
+    /// It will pretend that all vector sizes are equal to 10, all key value counts are equal to 10,
```

Comment:
>10 [](start = 59, length = 2)

Is there any specific reason for choosing 10?...:)

Reply:
No

---
In reply to: [218248145](https://github.com/dotnet/machinelearning/pull/931#discussion_r218248145) [](ancestors = 218248145)

Reply:
It should be more than one, that's the only limitation I'm aware of.

---
In reply to: [218253037](https://github.com/dotnet/machinelearning/pull/931#discussion_r218253037) [](ancestors = 218253037,218248145)